src/Compilers/CSharp/Portable/CommandLine/CommandLineParser.cs

```diff
@@ -1038,7 +1038,8 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
                 languageVersion: languageVersion,
                 preprocessorSymbols: defines.ToImmutableAndFree(),
                 documentationMode: parseDocumentationComments ? DocumentationMode.Diagnose : DocumentationMode.None,
-                kind: SourceCodeKind.Regular
+                kind: SourceCodeKind.Regular,
+                features: features.ToImmutableDictionary(feature => feature, feature => "true")
```

Comment:
Actually, you should look for an `=` in the string, so that you can set `/feature:foo=bar`. You should only use `true` when there is no `=`.